### ISSUE TITLE
alt text is not read as element isn't focusable [Global]

### ISSUE THREAD

**Issue by JuliannaR** [cds-snc/covid-shield-mobile#330]
**Scenario to reproduce**

All images in the app are not focusable. With this behaviour TalkBack and VoiceOver are unable to access the alt descriptions for the images. 

**Expected result**

To be read the alternative text for images.

**Bug or issue / WCAG**

1.1.1 Non text content
https://www.w3.org/TR/WCAG21/#non-text-content

**Screenshots where available**

![Image from iOS (1)](https://user-images.githubusercontent.com/35537308/86165348-2aff3e00-bae1-11ea-9ed3-3a178fe508c2.png)

**Device + OS**

iPhone 8 | 13.5.1

**Comment by its-anirudh** [comment]
hi @JuliannaR  , what should the voice over text say ? I am providing the text that's same as the main Title e.g.g ("How COVID alert works")

**Comment by JuliannaR** [comment]
@its-anirudh 

There is defined alt text for each image in the app. They are in the figma file.

EN: A hand-drawn illustration of a mobile phone in a pocket, sending and receiving signals.
FR: Dessin à la main d’un téléphone mobile dans une poche, qui envoie et reçoit des signaux.

Using the heading text doesnt give context and these images are not decorative, they convey meaning.

**Comment by JuliannaR** [comment]
@its-anirudh The alt text issue impacts all images throughout the app screens in the onboarding and how it works flows.

**Comment by its-anirudh** [comment]
@JuliannaR Added the alt text for How-to, and a potential fix for consistent VO navigation in the latest commit.

**Comment by JuliannaR** [comment]
+1 Thanks @its-anirudh

**Comment by JuliannaR** [comment]
fixed in V13.
